.github/workflows/cd-publish.yml

```diff
@@ -42,5 +42,5 @@ jobs:
    
     - name: Publish NPM changed packages
       run: |
-        yarn publish-ci -y  --access public $NPM_PAT 
+        yarn publish-ci -y -r --access public $NPM_PAT 
```

Comment:
Just want to make sure that `$NPM_PAT` isn't a typo.

Reply:
Oh, probably "public access token". I thought it was a typo for "path". Ignore me.

Reply:
sounds like a typo. it is without the $. but we need $ in CI. some people use ${NPM_PAT}, which does't work. So far $NPM_PAT is working for us.

Reply:
`-r` is default, so it should not be needed.

Reply:
I'm not seeing that NPM_PAT has been updated for read/write permissions.  Last update was October.  
![Screen Shot 2021-04-12 at 10 29 57 AM](https://user-images.githubusercontent.com/16669785/114436481-37d5b580-9b7a-11eb-90c4-e888df438693.png)

Reply:
let updated

Reply:
we can replace -r with "https://registry.npmjs.org/:_authToken=$NPM_PAT." we just can't put it in .npmrc for security reason.

Reply:
You can leave the code as is with `-r` then as I agree it shouldn't be in .npmrc.  Thanks for the clarification.